main.cpp

```diff
@@ -278,12 +278,13 @@ int main (int argc, char** argv)
 				std::cerr << "Error dispatching: " << retval << std::endl;
 			else if (retval != 0)
 				packets_read = true;
+			else
```

Comment:
Ah, that is indeed non-obvious. So for cleaning up old connections we use the time from the packets, where for refreshing the page we use the time from `gettimeofday`.

That seems like a good reason to keep the two separate, can we do that?